src/main/java/net/fabricmc/loom/configuration/ide/RunConfigSettings.java

```diff
@@ -160,11 +173,11 @@ public void setEnvironment(String environment) {
 	}
 
 	public String getConfigName() {
-		return name;
+		return configName;
 	}
 
 	public void setConfigName(String name) {
```

Comment:
Should it also have a `configName(String)` method to match the rest? I don't personally think it's needed, but it could be good for completeness. I guess it'd go away in 2.0 if this class moves to lazy properties though.

Reply:
Good question, for completeness yes, but for the sake of it being removed in 2.0 I dont think its needed.

Reply:
Just thinking about this, it should really be a display name, it feels a little weird deprecating this to something we arent 100% happy with.

Reply:
Well, it could be `Property<String> getDisplayName()` instead? With the old methods delegating to the property.

Reply:
Thats easier said than done, the default name is set in RunConfig, I think for now ill just do the easy change to make this use an object factory without breaking or deprecating anything. I really just want to get 1.3 done.